extractor/src/producer_class.py

```diff
@@ -48,6 +51,11 @@ def produce_event(self, topic, event):
             self.producer.poll(1)  # Callback function
         except ValueError:
             logger.error(format_traceback())
+        except KafkaException as e:
+            if e.args[0].code() == KafkaError.UNKNOWN_TOPIC_OR_PART:
+                pass
```

Comment:
Don't we want to at least log something here too?

Reply:
I'm ignoring this because It's expected when we cancel a batch (the topic is deleted).

Reply:
Oh, you're right!